app/services/pii/attributes.rb

```diff
@@ -2,7 +2,7 @@ module Pii
   Attributes = Struct.new(
     :first_name, :middle_name, :last_name,
     :address1, :address2, :city, :state, :zipcode,
-    :ssn, :dob, :phone,
+    :ssn, :dob, :phone, :otp,
```

Comment:
we're taking advantage of the fact that PII is serialized *and encrypted* on the as-yet-unverified profile. Otherwise we'd need to have a separate column on Profile, encrypted with the user's password, just for the OTP.